src/Sylius/Component/Taxonomy/spec/Model/TaxonSpec.php

```diff
@@ -130,7 +130,8 @@ function its_fullname_equal_name_if_no_parent(): void
 
     function its_fullname_prepends_with_parents_fullname(TaxonInterface $root): void
     {
-        $root->getFullname()->willReturn('Category');
```

Comment:
I would suggest to keep the original test (a call with no arguments), but add the second one as you did.

Reply:
Changed